fpdf/fonts.py

```diff
@@ -17,12 +17,18 @@
 from collections import defaultdict
 from dataclasses import dataclass, replace
 from functools import lru_cache
-from typing import Optional, Tuple, Union
+from typing import Optional, Tuple, Union, TYPE_CHECKING
 
 from fontTools import ttLib
 from fontTools.pens.ttGlyphPen import TTGlyphPen
 from fontTools.varLib import instancer
 
+if TYPE_CHECKING:  # Help static type checkers / language servers locate optional deps
+    try:  # pragma: no cover - typing-only
+        import uharfbuzz as hb  # type: ignore
+    except Exception:
+        hb = None  # type: ignore
+
```

Comment:
I don't understand why you added this block of code, since right below the code already does the exact same thing (import uharfbuzz as hb or default to None if there is an error)